flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/AbstractFileSystemTable.java

```diff
@@ -43,11 +43,10 @@
 
     List<String> partitionKeys;
 
-    AbstractFileSystemTable(DynamicTableFactory.Context context) {
+    AbstractFileSystemTable(DynamicTableFactory.Context context, ReadableConfig config) {
```

Comment:
do we still need the `Context` as an argument? can we simply pass the objects that we need? I don't think it is bad to list what is actually used in the constructor.